modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/history/util/CTCollectionTimelineUtil.java

```diff
@@ -17,6 +17,13 @@
  */
 public class CTCollectionTimelineUtil {
 
+	public static void setClassName(
+		HttpServletRequest httpServletRequest, Class<?> clazz) {
```

Comment:
the renderRequest method should be calling the httpServletRequest one incase the logic ever gets more complicated